examples/flutter_gallery/lib/demo/typography_demo.dart

```diff
@@ -49,15 +49,15 @@ class TypographyDemo extends StatelessWidget {
     final TextTheme textTheme = Theme.of(context).textTheme;
     final List<Widget> styleItems = <Widget>[
       if (MediaQuery.of(context).size.width > 500.0)
-        TextStyleItem(name: 'Display 4', style: textTheme.display4, text: 'Light 112sp'),
-      TextStyleItem(name: 'Display 3', style: textTheme.display3, text: 'Regular 56sp'),
-      TextStyleItem(name: 'Display 2', style: textTheme.display2, text: 'Regular 45sp'),
-      TextStyleItem(name: 'Display 1', style: textTheme.display1, text: 'Regular 34sp'),
-      TextStyleItem(name: 'Headline', style: textTheme.headline, text: 'Regular 24sp'),
-      TextStyleItem(name: 'Title', style: textTheme.title, text: 'Medium 20sp'),
-      TextStyleItem(name: 'Subheading', style: textTheme.subhead, text: 'Regular 16sp'),
-      TextStyleItem(name: 'Body 2', style: textTheme.body2, text: 'Medium 14sp'),
-      TextStyleItem(name: 'Body 1', style: textTheme.body1, text: 'Regular 14sp'),
+        TextStyleItem(name: 'Headline 1', style: textTheme.headline1, text: 'Light 112sp'),
+      TextStyleItem(name: 'Headline 2', style: textTheme.headline2, text: 'Regular 56sp'),
+      TextStyleItem(name: 'Headline 3', style: textTheme.headline3, text: 'Regular 45sp'),
+      TextStyleItem(name: 'Headline 4', style: textTheme.headline4, text: 'Regular 34sp'),
+      TextStyleItem(name: 'Headline 5', style: textTheme.headline5, text: 'Regular 24sp'),
+      TextStyleItem(name: 'Headline 6', style: textTheme.headline6, text: 'Medium 20sp'),
+      TextStyleItem(name: 'Subtitle 1', style: textTheme.subtitle1, text: 'Regular 16sp'),
```

Comment:
Is `Subtitle 2` missing here? I don't know if it is necessary, I noticed `overline` is also omitted.

Reply:
Yes, both of those styles are missing. This is just a simple translation of the existing demo - only the names have changed - I'm going to update to the demo to show the complete set of 2018 text styles in a separate PR.

https://github.com/flutter/flutter/pull/48547#discussion_r366971531